### DISCUSSION TITLE
Upload files and assign to a virtual folder

### DISCUSSION THREAD

**Discussion by Tymo93** [directus/directus#4170] [Feature Requests]
When uploading a file/image from app give option in which virtual folder to put it in.

My use case:
I want to make public images and files but not all of them only ones in a specific folder. This is currently possible using permissions but it is a chore to move each picture upload in to the respective folder.

Apologies if this has already been discussed or requested, I couldn't find anything about it here on github.

**Comment by rijkvanzanten** [comment]
Would https://github.com/directus/directus/pull/3209 fit your use case?

**Reply by Tymo93** [reply]
yes that is exactly what I need! I will subscribe to that thank you!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
